apps/strapi-cms/src/api/guide/content-types/guide/schema.json

```
@@ -0,0 +1,91 @@
+{
+  "kind": "collectionType",
+  "collectionName": "guides",
+  "info": {
+    "singularName": "guide",
+    "pluralName": "guides",
+    "displayName": "Guide",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
+  },
+  "attributes": {
+    "title": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string",
+      "required": true
+    },
+    "slug": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string",
+      "required": true
+    },
+    "listItems": {
+      "displayName": "ListItem",
+      "type": "component",
+      "repeatable": false,
```

```suggestion
      "repeatable": true,
```
In `apps/strapi-cms/src/api/guide/content-types/guide/lifecycles.ts` for this attribute should also be cheked that listItem is an Array of max 4 element. I check but unfortunately strapi does not permit to set it from component setting.